nixos/modules/programs/miriway.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
```

Comment:
Avoid these globally scoped `with`, they make codes hard to read.

Reply:
I do agree with you, but I think this might need a treewide fix. That part was copied verbatim from Sway's module for consistency, and most modules I checked unfortunately use a global `with lib`.